doc/ansible/dev/retroactively-owned-resources.md

```diff
@@ -0,0 +1,194 @@
+# Adding Owner References for Existing Resources
+
+Owner references are automatically injected `only during creation of
+resources`. Enabling owner reference injection `will not update objects`
+created while [owner reference injection is
+disabled](./advanced_options.md#turning-off-dependent-watches-and-owner-reference-injection)
+
+This guide will demonstrate how to retroactively set owner references
+for existing resources.
+
+A GET request to the owning resource will provide the necessary data to
+construct an `ownerReference` or an `annotation`.
+
+`$ kubectl get memcacheds.cache.example.com -o yaml`
+
+**Example Response (Abbreviated):**
+
+```yaml
+apiVersion: cache.example.com/v1alpha1
+kind: Memcached
+metadata:
+  name: example-memcached
+  namespace: default
+  uid: 2a94ff2b-84e0-40ce-8b5e-2b7e4d2bc0e2
+```
+
+`kubectl edit` can be used to update the resources by hand. See below
+for example `ownerReference` and `annotations`.
+
+## For objects in the same namespace as the Owner (CRD)
+
+Dependent resources `within the same namespace as the owning CR` are
+tracked with the `ownerReference` field.
+
+`ownerReference` structure:
+  * apiVersion: {group}/{version}
+  * kind: {kind}
+  * name: {metadata.name}
+  * uid: {metadata.uid}
+
+**Example ownerReference:**
+
+```yaml
+metadata:
+  ...(snip)
+  ownerReferences:
+    - apiVersion: cache.example.com/v1alpha1
+      kind: Memcached
+      name: example-memcached
+      uid: ad834522-d9a5-4841-beac-991ff3798c00
+```
+
+## For objects which are NOT in the same namespace as the Owner (CRD)
+
+An `annotation` is used instead of an `ownerReference` if the dependent
+resource is in a different namespace than the CR, or the dependent
+resource is a cluster level resource.
+
+`annotation` structure:
+  * operator-sdk/primary-resource: {metadata.namespace}/{metadata.name}
+  * operator-sdk/primary-resource-type: {kind}.{group}
+
+**NOTE**: The {group} can be found by splitting the `apiVersion`
+metadata of the CR, into `group` and `version`. As an example, 
+[this apiVersion field](https://github.com/operator-framework/operator-sdk-samples/blob/master/ansible/memcached-operator/deploy/crds/cache_v1alpha1_memcached_cr.yaml#L1)
+gives us the group `cache.example.com`.
+
+**Example Annotation:**
+
+```yaml
+metadata:
+  ...(snip)
+  annotations:
+    operator-sdk/primary-resource: default/example-memcached
+    operator-sdk/primary-resource-type: Memcached.cache.example.com
+```
+
```

Comment:
```suggestion
WDYT about just a note as, for example, something as:

**NOTES:** 
- The `operator-sdk/primary-resource` is formed by `<namespace>/<cr-name>.
- The `operator-sdk/primary-resource-type` is the name of CRD. 
```

Reply:
I think this note would be redundant with the [annotation structure](https://github.com/operator-framework/operator-sdk/pull/2013/files#diff-ffeb7f915e27543fa4033b94bcf6f6c3R59-R61)

Reply:
Make sense @asmacdo 👍